index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Portfolio</title>
+        <link rel="stylesheet" href="styles.css" ></style>
+    </head>
+    <body>
+        <div class="phone-width">
+            <div class="app-bar">
+                <div class="bar">
+                    
+                </div>
+                <header class="toolbar">
+                    <div class="logo">
+                        <p>
+                            John Doe
+                        </p>
+                    </div>
+                    <menu class="union-vector">
+                        <img src="img\Normal Button\Tertiary\Icons\Union.png" alt="menu button">
+                    </menu>
+                </header>
+            </div>
+            <div class="headline">
+                
+                <p class="primary-text">Hey There. </br> I'm Alex</p>
+                
+                <p class="supporting-text">
+                    I'am a Software Developer
+                </p>
```

Comment:
- [ ] Please use `hx` tag like `h2, h3 h4` for subtitle in this way will be semantically correct